docs/samples/Microsoft.ML.Samples/Dynamic/Trainers/BinaryClassification/Calibrators/Platt.cs

```diff
@@ -54,18 +53,30 @@ public static void Example()
             // Score   5.36571   Probability 0.9065308
         }
 
-        private static void PrintRowViewValues(Microsoft.ML.Data.DataDebuggerPreview data)
+        private static void PrintRowViewValues(IDataView transformedData)
```

Comment:
There are four similar functions. Would it be better to use TT file?

Reply:
Agreed, TT file would be ideal not just for these functions but for all four samples. It should be a task of its own and I'll open one. This particular issue addresses preview API usage. The reason I open a different task is because next few days the focus will be on documentation and I don't know if it make sense to prioritize rewriting these four samples with TT files. I'll still try to create TT files if I find time between documentation work. Thanks, @wschin  and let me know your thoughts.

Reply:
Spoke with @wschin offline and he is fine with my above proposal.